exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/TableBuilder.java

```diff
@@ -33,62 +33,91 @@
   private int w = 0;
   private int width;
 
-  public TableBuilder(final String[] columns) {
+  public TableBuilder(final String[] columns, final String[] columnTooltip) {
     sb = new StringBuilder();
     width = columns.length;
 
     format.setMaximumFractionDigits(3);
 
     sb.append("<table class=\"table table-bordered text-right\">\n<tr>");
-    for (final String cn : columns) {
-      sb.append("<th>" + cn + "</th>");
+    for (int i = 0; i < columns.length; i++) {
+      String cn = columns[i];
+      String ctt = "";
+      if (columnTooltip != null) {
+        String tooltip = columnTooltip[i];
+        if (tooltip != null) {
+          ctt = " title=\""+tooltip+"\" style=\"color:#00008B;\"";
```

Comment:
Since I'll be adding tooltips for all the columns in the 2 overview tables, I'll do away with the style